src/Middleware/HeaderPropagation/src/DependencyInjection/HeaderPropagationHttpClientBuilderExtensions.cs

```diff
@@ -22,7 +23,48 @@ public static IHttpClientBuilder AddHeaderPropagation(this IHttpClientBuilder bu
 
             builder.Services.AddHeaderPropagation();
 
-            builder.AddHttpMessageHandler<HeaderPropagationMessageHandler>();
+            builder.AddHttpMessageHandler(services =>
+            {
+                var options = new HeaderPropagationClientOptions();
+                var middlewareOptions = services.GetRequiredService<IOptions<HeaderPropagationOptions>>();
+                for (int i = 0; i < middlewareOptions.Value.Headers.Count; i++)
```

Comment:
use var.

Reply:
Is there a way to enable Visual Studio to highlight when certain code fixes are available? (I'm too used to ReSharper at work)